CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## main / unreleased
-
+* [ENHANCEMENT] Add timezone information to Alpine Docker images 
```

Comment:
Could you move this to the `### Grafana Mimir ` section near the other `[ENHANCEMENT]` entries?